gpt_index/data_structs/node_v2.py

```diff
@@ -94,8 +94,9 @@ def next_node_id(self) -> str:
     def get_text(self) -> str:
         """Get text."""
         text = super().get_text()
+        extra_info_exists = self.extra_info is not None and len(self.extra_info) > 0
         result_text = (
-            text if self.extra_info_str is None else f"{self.extra_info_str}\n\n{text}"
+            text if not extra_info_exists else f"{self.extra_info_str}\n\n{text}"
```

Comment:
NOTE: was injecting two newlines even when extra_info was an empty dict. thought it would make sense to change logic here